vulnerabilities/tests/test_redhat_importer.py

```diff
@@ -65,132 +67,81 @@ def test_to_advisory(self):
                         namespace="redhat",
                         name="bash",
                         version="4.2.46-28.el7",
-                        qualifiers=OrderedDict(),
+                        qualifiers={},
                         subpath=None,
                     ),
                     PackageURL(
                         type="rpm",
                         namespace="redhat",
                         name="bash",
                         version="4.1.2-48.el6",
-                        qualifiers=OrderedDict(),
```

Comment:
Feel free to remove OrderedDict from the whole codebase BTW since the dicts are always ordered now.